scss/_patterns_code-snippet.scss

```diff
@@ -1,15 +1,19 @@
 @import 'settings';
 
+$snippet-heading-bg-color: rgba($color-x-dark, 0.08);
```

Comment:
A bit nit picking, but our color variables usually start with `$color-`, so may be worth for consistency.

Reply:
Updated